### PR TITLE
Added Address Get with all info

### PR DESCRIPTION
**Pull request by DavidSKoppel**
In order to make it I had to also make one for postalcode

### CODE REVIEW

File: API/Controllers/AddressesController.cs [hunk 1]
```diff
@@ -55,96 +55,123 @@ public async Task<IActionResult> GetAddresses()
                     Id = address.Id,
                     StreetName = address.StreetName,
                     HouseNumber = address.HouseNumber,
-                    Postal = (int)address.PostalCodeId,
+                    //PostalCodeDto = (int)address.PostalCodeId,
```
Comment (Anders164a) [inline API/Controllers/AddressesController.cs:58]: Shouldn't it show/store Postal Code in address?
Or if the method is only to show the address, then remove commented code

**Review by Anders164a — COMMENTED** [review]
See comments

File: API/Controllers/AddressesController.cs (hunk 1 above)
Comment (DavidSKoppel) [reply API/Controllers/AddressesController.cs:58]: It seems to be a common mistake for all the GetBlanks to have unnecessary dtos

**Review by Anders164a — APPROVED** [review]
🚀

**Pull request merged**